src/components/DayPicker.jsx

```diff
@@ -102,6 +103,7 @@ export const defaultProps = {
   daySize: DAY_SIZE,
   isRTL: false,
   verticalHeight: null,
+  transitionDuration: 0.2,
```

Comment:
also here; if this is `undefined`, it will delegate to `CalendarMonthGrid` rather than duplicating the default prop value here